src/main/java/games/strategy/engine/lobby/server/login/LobbyLoginValidator.java

```diff
@@ -149,12 +159,12 @@ private String verifyConnectionInternal(final Map<String, String> propertiesSent
       return USERNAME_HAS_BEEN_BANNED + " " + getBanDurationBreakdown(usernameBanned.getSecond());
     }
     if (propertiesReadFromClient.containsKey(REGISTER_NEW_USER_KEY)) {
-      return createUser(propertiesSentToClient, propertiesReadFromClient, clientName);
```

Comment:
`propertiesSentToClient` is simply unused?

Reply:
Yes, it is no longer needed to process the authentication response (and it was prompted by my static analyzer).  It was previously used to get the unique public key for this particular client (sent in the authentication challenge).  Now that there is a single public/private key pair for the entire lobby server session, there is no need for the additional context.